civictechprojects/static/css/partials/_ProjectCard.scss

```diff
@@ -94,50 +95,3 @@
   padding: 0.25rem 0.5rem;
   margin: 0.25rem 0.5rem 0.25rem 0;
```

Comment:
this should be `.ProjectCard-title h2` in order to work properly (and the values should be changed to have 0.5 rem margin or padding, not both, top/right/bottom (left is provided by the grid layout having a spacer column, but if that's removed just make it margin: 0.5rem; or whatever)

to avoid this text hard up on the right edge: 
![image](https://user-images.githubusercontent.com/16870466/76591839-b612e480-64ae-11ea-836f-3bc226269adf.png)

Reply:
I don't know why I attached that comment to the wrong line, but I fixed this in 9de94c7